Capstone/Like View/LikeCell.h

```diff
@@ -0,0 +1,21 @@
+//
+//  LikeCell.h
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/12/22.
+//
+
+#import <UIKit/UIKit.h>
+#import "College.h"
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LikeCell : UITableViewCell
+@property (strong, nonatomic) College *college;
```

Comment:
Tip! If you have something like this:
```suggestion
@property (strong, readwrite, nonatomic) College *college;
```

In the .m file, you'll get a setter method for free. In that setter method, you can make all the UI updates:
```
- (void)setCollege:(College *)college {
   self.college = college;
   [self buildLikeCell];
}
```

That way, in LikeViewController, you won't need to call `[self buildLikeCell`] — you can just set the college to the cell, and the UI updates will happen for free